SportHub/package.json

```diff
@@ -61,7 +64,6 @@
   },
   "devDependencies": {
     "eslint-config-react-app": "^7.0.1",
-    "jest-editor-support": "^31.0.1",
-    "sass": "^1.62.1"
```

Comment:
sass why we need to remove that?

Reply:
It seems to work without all deleted dependencies. Removed them because some of them are considered to cause previous issue with installing packages